src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileAccessCategory.scala

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.stage.fileitem
+
+import nl.knaw.dans.common.lang.dataset.AccessCategory
+import nl.knaw.dans.common.lang.dataset.AccessCategory._
+
+object FileAccessCategory extends Enumeration {
+  type FileAccessCategory = Value
+  val ANONYMOUS, KNOWN, RESTRICTED_REQUEST, RESTRICTED_GROUP, NONE = Value
+
```

Comment:
I'm afraid the keyword mapping and its alternative doesn't allow a unit test that guarantees an exhaustive key-set.